FlowCal/excel_ui.py

```diff
@@ -133,7 +141,8 @@ def read_table(filename, sheetname, index_col=None):
 
     """
     # Catch sheetname as list or None
-    if sheetname is None or hasattr(sheetname, '__iter__'):
+    if sheetname is None or \
+        (hasattr(sheetname, '__iter__') and not isinstance(sheetname, str)):
```

Comment:
`basestring` doesn't exist in python 3. Conversion tools plainly replace `basestring` for `str`, so that's what I did. I can look into `six`.